### ISSUE TITLE
dlib/19.1

### ISSUE THREAD

**Issue by grisevg** [conan-io/wishlist#34]
dlib is a brilliant library for cutting edge computer vision, machine learning and image processing
https://github.com/davisking/dlib

**Comment by MojaveWastelander** [comment]
It doesn't seem to have a compex cmakefile, though have some dependencies which will need to add as options, I'll try to make it.

**Comment by MojaveWastelander** [comment]
I've added a basic [package](https://conan.io/source/dlib/19.1.0/MojaveWastelander/stable). It uses defaults of original library, I added most options from cmakelists into conanfile as well. For Windows latest version will compile only on msvc 14 and above (in appveyor all other versions are filtered), for Linux I execute test_package on a VM and it worked as expected, but in travis is not working yet (so no prebuilt libraries). Options not processed in conanfile: CUDA, LAPAS and BLAS.
